Packs/AggregatedScripts/Scripts/DomainEnrichment/DomainEnrichment.py

```diff
@@ -0,0 +1,138 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+from AggregatedCommandApiModule import *
+
+
+def domain_enrichment_script(
+    domain_list,
+    external_enrichment: bool = False,
+    verbose: bool = False,
+    enrichment_brands: list[str] | None = None,
+    additional_fields: bool = False,
+) -> CommandResults:
+    """
+    Enriches Domain data with information from various integrations.
+
+    Args:
+        domain_list (list[str]): List of domains to enrich.
+        external_enrichment (bool): Whether to perform external enrichment.
+        verbose (bool): Whether to print verbose output.
+        enrichment_brands (list[str]): List of brands to enrich.
+        additional_fields (bool): Whether to include additional fields in the output.
+
+    Returns:
+        CommandResults: The results of the command.
+    """
+    demisto.debug("Extracting indicators")
+    domain_list = extract_indicators(domain_list, "domain")
+    demisto.debug(f"Data list after extract_indicators: {domain_list}")
+    # Mapping for the final indicator objects (what you want to surface on each result)
+    indicator_mapping = {
+        "Name": "Name",
+        "DetectionEngines": "DetectionEngines",
+        "PositiveDetections": "PositiveDetections",
+        "Score": "Score",
+        "Brand": "Brand",
+    }
+
+    domain_indicator = Indicator(
+        type="domain",
+        value_field="Name",
+        context_path_prefix="Domain",
+        context_output_mapping=indicator_mapping,
+    )
+
+    # --- Command Batch 1: create indicators (BUILTIN) ---
+    demisto.debug("Command Batch 1: Creating new indicators")
+    command_batch1: list[Command] = [
+        Command(
+            name="CreateNewIndicatorsOnly",
+            args={"indicator_values": domain_list, "type": "Domain"},
+            command_type=CommandType.BUILTIN,
+            context_output_mapping=None,
+            ignore_using_brand=True,
+        )
+    ]
+
+    # --- Command Batch 2: internal analytics + external enrichment ---
+    command_batch2: list[Command] = []
+    if is_xsiam():
```

Comment:
can we use the same mechanism to check if core brand is enabled, instead of is_xsiam? We shouldnt treat as a special case if we dont need to. Same comment for IP